charter/index.html

```diff
@@ -291,92 +291,150 @@ <h1 id="title">
-      to assess consensus. The call for a vote must specify the duration of the
-      vote which must be at least 7 days and should be no more than 14 days.
-      The Chair must start the vote within 7 days of the request. The decision
-      will be based on the majority of the ballots cast. It is the Chair’s
-      responsibility to ensure that the decision process is fair, respects the
-      consensus of the CG, and does not unreasonably favor or discriminate
-      against any group participant or their employer.
-    </p>
+  
+  <section>
+    <h2>Decision Process</h2>
+    <p>This group will seek to make decisions when there is consensus. When
+    the group discusses an issue on the mailing or Issues list and there is a
+    call from the group for assessing consensus, after due consideration of
+    different opinions, the Chair should record a decision and any
+    objections.</p>
+    
+    <p>A common way to determine consensus for important decisions is to
+    conduct a Call for Consenus (CfC) where the Chair puts a proposal to the
+    group on the public mail list and asks for feedback from the Participants
+    within some period of time that is at least 7 days. Not responding
+    indicates going along with the proposal. Direct feedback is encouraged,
+    especially to weigh the degree of consensus when there is dissent.</p>
+    
+    <p>Participants MAY call for an online vote if they feel the Chair has
+    not accurately determined the consensus of the group or if the Chair
+    refuses to assess consensus. This should be a rare event, only where the
+    usual, less formal means of making decisions are not accepted. At least 5
+    Participants, no two from the same organization (or 50% of the
+    organizations and individuals, whichever is smaller), MUST agree with the
+    call for a formal vote. The call for a vote MUST specify the duration of
+    the vote which MUST be at least 7 days and should be no more than 14
+    days. The Chair MUST start the vote within 7 days of the request, or
+    group members MAY start it themselves. The decision will be based on the
+    majority of the ballots cast. It is the Chair's responsibility to ensure
+    that the decision process is fair, respects the consensus of the CG, and
+    does not unreasonably favor or discriminate against any group participant
+    or their employer.</p>
   </section>
-  <section><h2>Transparency</h2>
-    <p>
-      The group will conduct all of its technical work on its public mailing
-      list. Any decisions reached at any meeting are tentative. Any group
-      participant may object to a decision reached at an online meeting within
-      7 days of publication of the decision on the mail list. That decision must
-      then be confirmed on the mail list by the Decision Process above.
-    </p>
+  
+  <section>
+    <h2>Transparency</h2>
+    <p>The group will conduct all of its technical work on its The group will
```

Comment:
I did a PR to fix it.  The original sentence was from the template where all technical work on the spec was on the public mail list, not GitHub.